sdks/jreleaser-bluesky-java-sdk/src/main/java/org/jreleaser/sdk/bluesky/BlueskySdk.java

```diff
@@ -0,0 +1,169 @@
+ * limitations under the License.
+ */
+package org.jreleaser.sdk.bluesky;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import feign.form.FormEncoder;
+import feign.jackson.JacksonDecoder;
+import feign.jackson.JacksonEncoder;
+import org.jreleaser.bundle.RB;
+import org.jreleaser.logging.JReleaserLogger;
+import org.jreleaser.sdk.bluesky.api.BlueskyAPI;
+import org.jreleaser.sdk.bluesky.api.CreateSessionRequest;
+import org.jreleaser.sdk.bluesky.api.CreateSessionResponse;
+import org.jreleaser.sdk.bluesky.api.CreateTextRecordRequest;
+import org.jreleaser.sdk.commons.ClientUtils;
+import org.jreleaser.sdk.commons.RestAPIException;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+import static org.jreleaser.util.StringUtils.requireNonBlank;
+
+/**
+ * @author Tom Cools
+ * @author Simon Verhoeven
+ * @since 1.7.0
+ */
+public class BlueskySdk {
+    private final JReleaserLogger logger;
+    private final BlueskyAPI api;
+    private final boolean dryrun;
+    private final String handle;
+    private final String password;
+
+    private BlueskySdk(JReleaserLogger logger,
+                       boolean dryrun,
+                       String host,
+                       String handle,
+                       String password,
+                       int connectTimeout,
+                       int readTimeout) {
+        this.logger = requireNonNull(logger, "'logger' must not be null");
+        requireNonBlank(host, "'host' must not be blank");
+        this.handle = requireNonBlank(handle, "'handle' must not be blank");
+        this.password = requireNonBlank(password, "'password' must not be blank");
+        this.dryrun = dryrun;
+
+        ObjectMapper objectMapper = new ObjectMapper()
+            .setPropertyNamingStrategy(PropertyNamingStrategies.LOWER_CAMEL_CASE)
+            .setSerializationInclusion(JsonInclude.Include.NON_NULL)
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+            .configure(SerializationFeature.INDENT_OUTPUT, true);
+
+        api = ClientUtils.builder(logger, connectTimeout, readTimeout)
+            .encoder(new FormEncoder(new JacksonEncoder(objectMapper)))
+            .decoder(new JacksonDecoder(objectMapper))
+            .target(BlueskyAPI.class, host);
+
+        this.logger.debug(RB.$("workflow.dryrun"), dryrun);
+    }
+
+    public void skeet(List<String> statuses) throws BlueskyException {
+        wrap(() -> {
+            CreateSessionRequest sessionRequest = CreateSessionRequest.of(handle, password);
+            CreateSessionResponse session = api.createSession(sessionRequest);
+
+            for (String status : statuses) {
```

Comment:
Does BlueSky support threads?

Reply:
There's indeed no way to predict how long threads may be. So far JReleaser has tweeted/tooted 14 status in a single thread.

Reply:
"Over the weekend of April 29-30th, 2023, as the hellthread expanded and roped in more and more users, Bluesky staff told users that it could not be fixed until Paul Frazzle, the person responsible for that aspect of the code, came into work on Monday, May 1st. Jay Graber and the team allowed the hellthread to flourish until at least the start of the work week.[[1]](https://knowyourmeme.com/memes/events/hellthread-hellrope-bluesky#fn1)"

Lol 😆 poor Paul

Reply:
@SimonVerhoeven let's keep this moving. I'll make the changes to post threads later today. Let's finish that up, you do another test run and then we can Squish Squash the commits to comply with the guidelines.

Reply:
We have implemented Threads in the latest push @aalmiray. Other than a merge request on the documentation, I think we are done here. :-) 

![image](https://github.com/jreleaser/jreleaser/assets/11362563/dec8b1ee-cfdc-451e-a7eb-5ec1ffad3202)